docs/probe.md

```diff
@@ -0,0 +1,177 @@
+# Probes and Meters: Examples for Adding to Code
```

Comment:
I think `probe.md` could use a better name, but not sure exactly what's right. This doc to me feels like "examples of using Armory's instrumentation feature"

Reply:
I'm actually starting to wonder if a better home for this kind of demo is in the `armory/notebooks` dir

Reply:
> I think `probe.md` could use a better name, but not sure exactly what's right. This doc to me feels like "examples of using Armory's instrumentation feature"

any thoughts on updates in 89d200a6?